### ISSUE TITLE
Emote seems not working here.

### ISSUE THREAD

**Issue by Boifuba** [League-of-Foundry-Developers/fvtt-module-trigger-happy#44]
I'm using @Drawing[bomdia]@ChatMessage[Bom Dia!]{Guarda}@Trigger[move capture emote]
But the Token called Guard is not talking through balloon. Am i missing something?

**Comment by tposney** [comment]
Can you try making the @fields @Drawing, @Trigger etc

**Comment by tposney** [comment]
I think the problem is that the chat message text needs to be in quotes.
```
@Drawing[bomdia]@ChatMessage["Bom Dia!"]{Guarda}@Trigger[move capture emote]
```

**Comment by p4535992** [comment]
should be solved on 0.8.21.
For the new logic the syntax must respect a certain order so your label now is:

`@drawing[bomdia]@trigger[move capture emote]@ChatMessage[Bom Dia!]{Guarda}`

let me know.
